lang/en.json

```diff
@@ -192,7 +192,7 @@
   "GURPS.hitLocationWhere": "Where",
   "GURPS.hitLocationPenalty": "Penalty",
   "GURPS.hitLocationDR": "DR",
-  "GURPS.hitLocationDRWithDivisor": "Effective DR",
```

Comment:
Why remove this line? I changed the text to "Effective DR" because after I added shotgun damage, this line showed the effects of applying armor divisor and/or the shotgun damage multiplier. So it should stay in the file.

Reply:
I removed that line by mistake. Re-added!